crypto/american-breakfast/README.md

```diff
@@ -0,0 +1,12 @@
+# American Breakfast
+
+This challenge is actually a custom bacon cipher. 
+
+Instead of A and B, I used T and R.
```

Comment:
Speak in a more third-person format. For example, "T and R are used instead of A and B".

Reply:
fixed